integrationtests/disk_v1alpha1_test.go

```diff
@@ -0,0 +1,94 @@
+package integrationtests
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/kubernetes-csi/csi-proxy/client/api/disk/v1alpha1"
+	diskv1alpha1client "github.com/kubernetes-csi/csi-proxy/client/groups/disk/v1alpha1"
+	"github.com/stretchr/testify/require"
+)
+
+func v1alpha1DiskTests(t *testing.T) {
+	t.Run("ListDiskIDs,ListDiskLocations", func(t *testing.T) {
+		// even though this test doesn't need the VHD API it failed in Github Actions
+		// see https://github.com/kubernetes-csi/csi-proxy/pull/140/checks?check_run_id=2671787129
```

Comment:
what is the reason it will fail?

Reply:
I think the permalink doesn't work anymore :(, the error is:

```
    disk_v1beta3_test.go:30: 
        	Error Trace:	disk_v1beta3_test.go:30
        	Error:      	Expected nil, but got: &status.statusError{state:impl.MessageState{NoUnkeyedLiterals:pragma.NoUnkeyedLiterals{}, DoNotCompare:pragma.DoNotCompare{}, DoNotCopy:pragma.DoNotCopy{}, atomicMessageInfo:(*impl.MessageInfo)(nil)}, sizeCache:0, unknownFields:[]uint8(nil), Code:2, Message:"Could not get page83 ID: IOCTL_STORAGE_QUERY_PROPERTY failed: Incorrect function.", Details:[]*anypb.Any(nil)}
        	Test:       	TestDiskAPIGroup/v1beta3Tests/ListDiskIDs,ListDiskLocations
```

This error is because the `ListDiskIDs` API and the page83 ID